ruleset/rules/0850-audit_rules.xml

```diff
@@ -1,83 +1,70 @@
-    <!-- Sample 2: type=SYSCALL msg=audit(1624643172.076:11843): arch=c000003e syscall=59 success=yes exit=0 a0=25c81f0 a1=255f000 a2=255df40 a3=7ffc66d5c8e0 items=3 ppid=4353 pid=13331 auid=1198400500 uid=1198400500 gid=1198400513 euid=1198400500 suid=1198400500 fsuid=1198400500 egid=1198400513 sgid=1198400513 fsgid=1198400513 tty=pts0 ses=48 comm="runtime" exe="/usr/bin/python2.7" subj=unconfined_u:unconfined_r:unconfined_t:s0-s0:c0.c1023 key="wazuh_execution"\u001DARCH=x86_64 SYSCALL=execve AUID="administrator@ExchangeTest.com" UID="administrator@ExchangeTest.com" GID=646F6D61696E2075736572734045786368616E6765546573742E636F6D EUID="administrator@ExchangeTest.com" SUID="administrator@ExchangeTest.com" FSUID="administrator@ExchangeTest.com" EGID=646F6D61696E2075736572734045786368616E6765546573742E636F6D SGID=646F6D61696E2075736572734045786368616E6765546573742E636F6D FSGID=646F6D61696E2075736572734045786368616E6765546573742E636F6D type=EXECVE msg=audit(1624643172.076:11843): argc=6 a0="/usr/bin/python3" a1="./runtime" a2="/tmp/calc.py" a3="exchangetest.com/administrator@192.168.0.57" a4="-hashes" a5="c615d74f277acb732af4e8680330fcf1:c615d74f277acb732af4e8680330fcf1" type=CWD msg=audit(1624643172.076:11843):  cwd="/tmp" type=PATH msg=audit(1624643172.076:11843): item=0 name="./runtime" inode=8409155 dev=fd:00 mode=0100755 ouid=1198400500 ogid=1198400513 rdev=00:00 obj=unconfined_u:object_r:user_tmp_t:s0 objtype=NORMAL cap_fp=0000000000000000 cap_fi=0000000000000000 cap_fe=0 cap_fver=0\u001DOUID="administrator@ExchangeTest.com" OGID=646F6D61696E2075736572734045786368616E6765546573742E636F6D type=PATH msg=audit(1624643172.076:11843): item=1 name="/usr/bin/python3" inode=1691048 dev=fd:00 mode=0100755 ouid=0 ogid=0 rdev=00:00 obj=system_u:object_r:bin_t:s0 objtype=NORMAL cap_fp=0000000000000000 cap_fi=0000000000000000 cap_fe=0 cap_fver=0\u001DOUID="root" OGID="root" type=PATH msg=audit(1624643172.076:11843): item=2 name="/lib64/ld-linux-x86-64.so.2" inode=4636927 dev=fd:00 mode=0100755 ouid=0 ogid=0 rdev=00:00 obj=system_u:object_r:ld_so_t:s0 objtype=NORMAL cap_fp=0000000000000000 cap_fi=0000000000000000 cap_fe=0 cap_fver=0\u001DOUID="root" OGID="root" type=PROCTITLE msg=audit(1624643172.076:11843): proctitle=2F7573722F62696E2F707974686F6E33002E2F72756E74696D65007073657865632E70790065786368616E6765746573742E636F6D2F61646D696E6973747261746F72403139322E3136382E302E3537002D6861736865730063363135643734663237376163623733326166346538363830333330666366313A633631356437 -->
-    <rule id="92101" level="6">
-        <if_sid>92100</if_sid>
-        <match type="pcre2">a\d="\w+\.py".+cwd="/tmp"|a\d="/tmp/.+\.py</match>
-        <description>Executed python script from /tmp/ folder</description>
-        <mitre>
-            <id>T1059.006</id>
-        </mitre>
-    </rule>
+  <rule id="92102" level="12">
+    <if_sid>92100, 92101</if_sid>
+    <match type="pcre2">psexec.py|smbexec.py</match>
+    <match type="pcre2">a\d="-hashes".+a\d=".+:.+</match>
```

Comment:
This match seems to be wrong, this is equivalent to:
```
psexec.py|smbexec.pya\d="-hashes".+a\d=".+:.+
```
I think something like this should have been added:
```
psexec.py|smbexec.py|a\d="-hashes".+a\d=".+:.+
```
Please add unitary cases of this rule.

Reply:
Actually the intention here is to have an "and" operation between the two match expressions